drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/common/DecisionTableColumnViewUtils.java

```diff
@@ -60,6 +77,51 @@ public static void addWidgetToContainer(final IsWidget widget,
                container);
     }
 
+    public static String getColumnManagementGroupTitle(final BaseColumn column) {
+        String managementSectionHeader = null;
+        if (column instanceof ActionInsertFactCol52) {
+            managementSectionHeader = concatenateFactTypeAndBoundName(((ActionInsertFactCol52) column).getFactType(),
+                                                                      ((ActionInsertFactCol52) column).getBoundName());
+        } else if (column instanceof ActionSetFieldCol52) {
+            managementSectionHeader = concatenateFactTypeAndBoundName(null,
+                                                                      ((ActionSetFieldCol52) column).getBoundName());
+        } else if (column instanceof ActionWorkItemInsertFactCol52) {
+            managementSectionHeader = concatenateFactTypeAndBoundName(((ActionWorkItemInsertFactCol52) column).getFactType(),
+                                                                      ((ActionWorkItemInsertFactCol52) column).getBoundName());
+        } else if (column instanceof ActionWorkItemSetFieldCol52) {
+            managementSectionHeader = concatenateFactTypeAndBoundName(null,
+                                                                      ((ActionWorkItemSetFieldCol52) column).getBoundName());
+        } else if (column instanceof Pattern52) {
```

Comment:
I think that we could remove the `if`s related to `ActionWorkItemInsertFactCol52` and `ActionWorkItemSetFieldCol52`, since they extends respectively `ActionInsertFactCol52` and `ActionSetFieldCol52`.